icekit/bin/setup-postgres.sh

```diff
@@ -10,6 +10,13 @@ EOF
 
 set -e
 
+# Is the `pv` utility available
+if [[ `pv --help > /dev/null 2>&1` ]]; then
+    PV_CMD=pv
+else
+    PV_CMD=cat
+fi
```

Comment:
Just doing `alias pv=cat` when pv is not available might be enough to work-around this usage, then we don't have to use `$PV_CMD` elsewhere in scripts?

Or we could just fail loudly in `entrypoint.sh` if `pv` (or any other dependencies) is not available, when run outside of docker.